scene/gui/rich_text_label.cpp

```diff
@@ -36,6 +36,7 @@
 #include "core/os/os.h"
 #include "core/string/translation.h"
 #include "label.h"
```

Comment:
Not really related to your PR, but since you're in the area, could you fix this to use an absolute path?